storybook-utilities/components/AdditionalInputInfo.js

```diff
@@ -0,0 +1,162 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import SprkList from '../../react/src/base/lists/SprkList';
+import SprkListItem from '../../react/src/base/lists/SprkListItem';
+import { getStorybookInstance } from '../getStorybookInstance';
+
+const AdditionalInputInfo = (props) => {
+  const {
+    additionalHeaderClasses,
+    additionalListClasses,
+    additionalListItemClasses,
+    headerElement,
+    listElement,
+    listVariant,
+  } = props;
+
+  const storybookInstance = getStorybookInstance();
+  const isHTMLStorybook = (storybookInstance === 'html');
+
+  const TagName = headerElement;
+  return (
+    <React.Fragment>
+      <TagName
+        className={additionalHeaderClasses}
+      >
+        Additional Input Information
+      </TagName>
+      <SprkList
+        element={listElement}
+        additionalClasses={additionalListClasses}
+        variant={listVariant}
+      >
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          By default, Spark does not provide validation or formatting.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          Special characters used in formatting (like the hyphens
+          in the SSN field) are not automatically removed from the field
+          value and may need to be manually removed.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          Each input element&apos;s id should be unique on the page.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          The
+          <code> type </code>
+          attribute for an Input should be the most
+          appropriate value that semantically matches your use case.
+        </SprkListItem>
+        <SprkListItem
+          additionalClasses={additionalListItemClasses}
+        >
+          The
+          <code> data-id </code>
+          property is provided as a hook for automated tools.
+          If you have multiple instances of the same variant
+          of a component on the same page, make sure each instance
+          has a unique
+          <code> data-id </code>
+          property (&quot;input-text-1&quot;,
+          &quot;input-text-2&quot;, &quot;input-date-1&quot;, etc).
+        </SprkListItem>
+        {isHTMLStorybook && (
+          <React.Fragment>
```

Comment:
You dont need to change this, but you can do `<>` instead of `<React.Fragment>` now.